management/dbaas/psmdb_cluster_test.go

```diff
@@ -187,7 +190,7 @@ func TestPSMDBClusterServer(t *testing.T) {
 		}
 		_, err := dbaasClient.Default.PSMDBCluster.CreatePSMDBCluster(&paramsPSMDBInvalidName)
 		assert.Error(t, err)
-		assert.Equal(t, 500, err.(pmmapitests.ErrorResponse).Code())
+		assert.Equal(t, 400, err.(pmmapitests.ErrorResponse).Code())
```

Comment:
Can we use `pmmapitests.AssertAPIErrorf` here? It's probably returning 400 because of empty disk size

Reply:
Added.